internal/graphql/resolver/meshsync.go

```diff
@@ -4,30 +4,33 @@ import (
 	"context"
 
 	"github.com/layer5io/meshery/internal/graphql/model"
+	"github.com/layer5io/meshkit/broker"
 	"github.com/layer5io/meshkit/database"
 	"github.com/layer5io/meshkit/utils"
 	mesherykube "github.com/layer5io/meshkit/utils/kubernetes"
-	"github.com/layer5io/meshsync/pkg/broker"
 	meshsyncmodel "github.com/layer5io/meshsync/pkg/model"
 )
 
 var (
-	meshsyncName = "meshsync"
-	meshsyncYaml = "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/samples/meshery_v1alpha1_meshsync.yaml"
+	meshsyncName    = "meshsync"
+	meshsyncSubject = "meshery.>"
+	meshsyncQueue   = "meshery"
+	meshsyncYaml    = "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/samples/meshery_v1alpha1_meshsync.yaml"
```

Comment:
Hmm. If we don't externalize now, it'd be good to create an issue to externalize this later.

Reply:
yes, rather actually move it as a `config`. Meshery is the only repo currently without a config.